Dockerfile

```diff
@@ -0,0 +1,23 @@
+#########################################################
+# Dockerfile for deploying storybook                    #
+# Usage example:                                        #
+# - Build image:                                        #
+# $ docker build . -t haoict/storybook-wido             #
```

Comment:
What those comments for? Can we remove that?

Reply:
Just simple examples how to build and run the images, for noobs

Reply:
yeah should remove that!